helm/coco-kafka-bridge/app-configs/kafka-bridge_eks_delivery_dev.yaml

```diff
@@ -19,9 +18,29 @@ bridges:
     replicas: 2
   # Cms metadata bridge to pub cluster
   - name: cms-metadata-kafka-bridge-pub
-    sourceKafkaProxyUrl: "https://upp-k8s-dev-publish-eu.upp.ft.com" # use the pub cluster url
+    sourceKafkaProxyUrl: "https://upp-k8s-dev-publish-eu.upp.ft.com/__kafka-rest-proxy" # use the pub cluster url
     groupIdPrefix: metadata-kafka-bridge-pub
     topic: NativeCmsMetadataPublicationEvents
+    producer: "http://kafka-rest-proxy:8080"
+    authSecretName: pub-auth
+    authSecretKey: pub-k8s-auth
+    type: "proxy"
+    replicas: 2
+  # Bridge to MSK
+  - name: kafka-to-msk-bridge
+    sourceKafkaProxyUrl: "http://kafka-rest-proxy:8080"
+    groupIdPrefix: kafka-msk-bridge
+    topic: CombinedPostPublicationEvents
+    producer: "http://kafka-rest-proxy-msk:8080"
+    authSecretName: pub-auth
+    authSecretKey: pub-k8s-auth
+    type: "proxy"
+    replicas: 2
+  - name: msk-to-kafka-bridge
+    sourceKafkaProxyUrl: "http://kafka-rest-proxy-msk:8080"
+    groupIdPrefix: kafka-msk-bridge
+    topic: NativeCmsPublicationEvents
+    producer: "http://kafka-rest-proxy:8080"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏This is needs [UPPSF-2718](https://financialtimes.atlassian.net/browse/UPPSF-2718) to be completed in order for this bridge to work properly right?

Reply:
It actually needs 2717 to be completed in order to enable 2718. That's why I added the config needed for 2718 here as well so we can iterate faster.